src/cache/cache_las.c

```diff
@@ -585,25 +615,24 @@ __wt_las_insert_updates(WT_CURSOR *cursor, WT_BTREE *btree, WT_PAGE *page, WT_MU
         full_value->data = upd->data;
         full_value->size = upd->size;
 
-        prev_upd = NULL;
         squashed = false;
 
-        /* Flush the updates on stack */
+        /* Flush the updates on stack. */
         for (; modifies.size > 0; tmp = full_value, full_value = prev_full_value,
                                   prev_full_value = tmp, upd = prev_upd) {
-            /* Should not see BIRTHMARK or TOMBSTONE */
+            /* Should not see BIRTHMARK or TOMBSTONE. */
```

Comment:
You could just remove this comment - it's just restating what the code clearly says. Similarly to `Skip TOMBSTONE` below.